### ISSUE TITLE
fas.li

### ISSUE THREAD

**Issue by ahmedk218** [reek/anti-adblock-killer#1762]
examble
http://fas.li/v6YmS

it say 

Please disable Adblock
is there a way not to disable the adblock

**Comment by tabish4448** [comment]
Yes there is. Use this [https://github.com/reek/anti-adblock-killer/issues/1725#issuecomment-231913672](url)

**Comment by tabish4448** [comment]
Yes there is. here: #1725

**Comment by maynak00** [comment]
http://fas.li/v6YmS > http://shink.in/wunaG > http://al.ly/tf9vt > http://ouo.io/eU9vY4 > http://46.105.92.222/sv3/?file=33f4c5fad2
fas.li, shink.in,ouo.io no antiadblock message for me.but al.ly blockadblock

**Comment by tabish4448** [comment]
@maynak00 Working fine here.

![1](https://cloud.githubusercontent.com/assets/20391995/16846339/791e5626-4a05-11e6-86ab-8f1fddf6a0ea.png)

![2](https://cloud.githubusercontent.com/assets/20391995/16846350/86c0559a-4a05-11e6-8d28-14eedd9d3dcd.png)

**Comment by tabish4448** [comment]
Apply this:
https://gist.github.com/reek/dc234ea9687056bd99fe (click raw for install Script)
https://gist.github.com/reek/292ed06433b4cfc07bc94cb4fec4adbe (copy raw url add it to AB)

by @reek #1725

**Comment by X01X012013** [comment]
I think they hide the real link on server side, the ad can be blocked, but can't really skip the screen.
